packages/react-core/src/components/Title/examples/TitleDefaultSizes.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Title } from '@patternfly/react-core';
+
+export const DefaultTitleSizes: React.FunctionComponent = () => (
```

Comment:
For the exported name, we should follow the convention of other TypeScript examples where the format is typically  `[componentName][exampleTitle]`.

```suggestion
export const TitleDefaultSizes: React.FunctionComponent = () => (
```